feat: add fullscreen relevant type definitions

otacke opened the pull request:
When merged in, will add the type definitions relevant for using fullscreen features.

boyum commented:
Thanks for the contribution! Please add this to the `H5PObject` file as well, and run `npm run package` after.

Also, do you have any thoughts about this repository? I didn't want to add it to DefinitelyTyped, but maybe that's the correct decision? I don't think the H5P maintainers wants to use it, but we prefer to use it in our projects, because it's a little more up to date than the official documentation.

otacke commented:
Sorry. Added that and fixed the wrong file name for the xAPI event type as an extra :-)

I usually don't use typescript if I don't have to, but for those who do, the type definitions surely come in handy.

boyum commented:
That's completely valid, of course! These type definitions will also work with JSDoc if you want to type check your JS files.

otacke commented:
I don't have a strong opinion on TypeScript :-) I have been working with it and simply felt it's not something that I like, but I fully understand if some people love it.
